### ISSUE TITLE
Mistake in a unit test (language translation)

### ISSUE THREAD

**Issue by kulshekhar** [udacity/deep-learning#72]
I'm not sure but [this line](https://github.com/udacity/deep-learning/blob/master/language-translation/problem_unittests.py#L156) in the unit test doesn't look right. The shape of `inf_output` isn't being tested.

```python
assert train_output.get_shape().as_list() == [batch_size, None, vocab_size],\
            'Train Logits is the wrong shape: {}'.format(train_output.get_shape())
assert train_output.get_shape().as_list() == [batch_size, None, vocab_size], \
            'Inference Logits is the wrong shape: {}'.format(train_output.get_shape())
```

**Comment by jg1141** [comment]
I see same mistake on lines 156-157 and again on lines 184-185. Test of train_output was copy/pasted and only the text of the error message was changed.

**Comment by Brok-Bucholtz** [comment]
We've fixed the issue with the training logits being checked instead of the inference logits:
https://github.com/udacity/deep-learning/pull/78

Thanks for letting us know!
